gef-ui/server/controllers/amendments/do-you-have-a-facility-end-date/get-do-you-have-a-facility-end-date.ts

```diff
@@ -48,10 +48,13 @@ export const getDoYouHaveAFacilityEndDate = async (req: GetDoYouHaveAFacilityEnd
       );
     }
 
+    const isUsingFacilityEndDate = amendment.isUsingFacilityEndDate === undefined ? undefined : amendment.isUsingFacilityEndDate.toString();
```

Comment:
```suggestion
    const isUsingFacilityEndDate = amendment.isUsingFacilityEndDate ?? amendment.isUsingFacilityEndDate.toString();
```

Do you think we can use nullish coalescing to simplify the conditional statement?

Reply:
I think thats the wrong way around - if `isUsingFacilityEndDate` is undefined we want to return `undefined` not `'undefined'`.

Agree this is a bit ugly but Im not sure of a simpler way to deal with it :/ 


Something like `amendment.isUsingFacilityEndDate && amendment.isUsingFacilityEndDate.toString();` doesnt work if `isUsingFacilityEndDate` is `false`

Reply:
In that case this will work.

```ts
amendment.isUsingFacilityEndDate ?? amendment.isUsingFacilityEndDate?.toString()
```